src/app/components/Grid/index.jsx

```diff
@@ -46,8 +102,275 @@ export const CPSPageGrid = ({ children, ...props }) => (
   </StyledCPSPageGrid>
 );
 
+export const ArticlePageGrid = ({ as, ...props }) => (
```

Comment:
Looks like `ArticlePageGrid` is the same as `GridWrapper` so I'd remove `ArticlePageGrid` and just use `GridWrapper`